Dockerfile

```diff
@@ -19,10 +19,10 @@ RUN apt-get update \
 
 RUN apt -y update
 
-ARG OPENSSL_VERSION=1.1.1l
+ARG OPENSSL_VERSION=1.1.1q
```

Comment:
The code was inherited this way. They choose to install from the source. I have no idea why.

Reply:
Yes, it's best practice to install using apt, because installing for source requires us to hardcode OpenSSL version in the dockerfile, and it can fall behind over time without being upgraded between rebuilds when new versions of OpenSSL become available.

Reply:
If we use apt to install openssl, do we still need to create a symbolic link to the /etc/ssl/certs or that will be taken care of during the install?

Reply:
Not sure. Could you please refer to the DLC dockerfiles where openssl is installed using apt?

Reply:
I couldn't find where symbolic link is created when openssl is installed using apt. So I will keep the code as is for now.